spec/producer_spec.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe FastlyNsq::Producer do
+  let!(:topic) { 'fnsq' }
```

Comment:
i did know `let!` is a thing and I love it.

Reply:
The reset is to ensure a clean state in nsq and to fix consistency issues I was having with the test suite.